fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -783,25 +784,43 @@ private void handleAnalyzeStmt() throws IOException {
             return;
         }
 
-        AnalyzeStatus analyzeStatus;
-        StatisticExecutor statisticExecutor = new StatisticExecutor();
+        Thread thread = new Thread(() -> {
```

Comment:
too many threads.

Reply:
If you don't want to manage a thread pool, you can use `CompletableFuture.runAsync`